cmd2/argparse_completer.py

```diff
@@ -209,6 +209,33 @@ def register_custom_actions(parser: argparse.ArgumentParser) -> None:
     parser.register('action', 'append', _AppendRangeAction)
 
 
+def token_resembles_flag(token: str, parser: argparse.ArgumentParser):
```

Comment:
This new function should specify a return value type hint of ``bool``.  

Also, unit tests for this function should be added